packages/hint-sri/src/meta.ts

```diff
@@ -15,13 +15,13 @@ const meta: HintMetadata = {
         additionalProperties: false,
         properties: {
             baseline: {
-                oneOf: Object.keys(Algorithms).filter((key) => {
+                enum: Object.keys(Algorithms).filter((key) => {
```

Comment:
Another rule that [does something similar is `performance-budget`](https://github.com/webhintio/hint/blob/64d0ed2357f34aac346ed20c689033ca266e510a/packages/hint-performance-budget/src/meta.ts#L18).
The code in this case is:

```ts
connectionType: {
    oneOf: [{ enum: Connections.ids }],
        type: 'string'
    },
```

RIght now I don't know which one is the "most correct" one 🤔
If you have time, do you mind doing a bit of digging? If not I'll research in the morning. Tomorrow should be a calmer day.

Thanks!

Reply:
As far as I understand the JSON schema rules the `performance-budget` schema is redundant. The following definition should be equivalent:

```javascript
{
    // ...
    connectionType: {
        enum: Connections.ids,
        type: 'string'
    },
    // ...
}
```

I think `oneOf` says: "one of the things that follow in the array". In the case of the `connectionType` property the array has only one member and therefore I think it means "one of one".

I tested this assumption and can confirm that the `performance-budget` hint can still be configured when the changes described above are made to its metadata definition.